build-info-extractor/src/test/java/org/jfrog/build/extractor/issuesCollection/IssuesCollectorTest.java

```diff
@@ -92,6 +92,13 @@ public void testCollectIssuesWithRevision() throws IOException, InterruptedExcep
         runCollectIssues("git_issues2_.git_suffix", vcsList, 2);
     }
 
+    @Test
+    // Test collection with a made up revision - the command should not throw an error, and 0 issues should be returned.
```

Comment:
Let's change the comment style to be 
```
/**

 */
```
Let's also add to the description the reason why we test this scenario.